virtool/samples/utils.py

```diff
@@ -52,6 +57,16 @@ def calculate_workflow_tags(analyses: list) -> dict:
     }
 
 
+async def check_labels(labels, postgres_db):
+    non_existent_labels = list()
+    for label_id in labels:
+        async with AsyncSession(postgres_db) as session:
```

Comment:
This is very inefficient. Can you find a method for accomplishing this in one query?